src/pages/Checklist.js

```diff
@@ -57,18 +62,18 @@ const refundRows = ({
   line484,
 } = {}) => {
   return [
-    { key: 'Total Income (150):', value: line150, id: 'line150' },
-    { key: 'Net Income (236):', value: line236, id: 'line236' },
-    { key: 'Taxable Income (260):', value: line260, id: 'line260' },
-    { key: 'Net Federal Tax (420):', value: line420, id: 'line420' },
-    { key: 'Net Ontario Tax (428):', value: line428, id: 'line428' },
-    { key: 'Total Payable (435):', value: line435, id: 'line435' },
-    { key: 'Total Deducted (437):', value: line437, id: 'line437' },
-    { key: 'Total Credits (482):', value: line482, id: 'line482' },
-    { key: 'Total payable minus credits:', value: '400', id: 'totalPayableMinus' },
-    { key: 'Previous balance owed:', value: '0', id: 'previousBalance' },
-    { key: 'Current balance owed:', value: '0', id: 'currentBalance' },
-    { key: 'Refund (484):', value: line484, id: 'line484' },
+    { key: 'Total Income (150)', value: line150, id: 'line150' },
```

Comment:
We only need an `id` for each of these rows if we're including the `.action` cell (ie, the "Change" link). 
If we're getting rid of it, we can remove the `id` entirely.

Reply:
Also, you could run all the values through the `currency` function. (Or probably there's another way to loop it).

Reply:
Maybe the user income information should just be strings. What do you think, @sboisvert?

Reply:
I ended up using `id:` for the row indentation but only on those rows, is that ok?